tests/keras/layers/test_call.py

```diff
@@ -0,0 +1,52 @@
+# encoding: utf-8
+"""Test seya.layers.recurrent module"""
+
+from __future__ import print_function
+
+import unittest
```

Comment:
@EderSantana, thank you very much for the test! keras uses pytest so I personally don't see any point of using unittest classes. Unittest is too "java-ish", pytest is more modern framework.